packages/pardi/pardi.2.0.2/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/pardi"
+bug-reports: "https://github.com/UnixJunkie/pardi/issues"
+dev-repo: "git+https://github.com/UnixJunkie/pardi.git"
+license: "GPL-1.0-or-later"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {>= "1.6"}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
This constraint is wrong according to your `dune-project` file. Could you fix this?

Reply:
Oh sorry, I thought I had checked it. We should start making this check automatic... I'll send a PR

Reply:
thank you both